app/assets/javascripts/main.js.erb

```diff
@@ -183,12 +183,12 @@ $(function() {
         icon: constants.priorityMarkerImage
       }, %>
       adopted: {
-        name: 'Adopted',
-        title: 'Adopted by other',
+        name: "%{c(brand.gis_icon_2)}",
+        title: "Adopted by other",
         icon: constants.adoptedMarkerImage
       },
       adopted_by_you: {
-        name: 'Adopted by you',
+        name: "%{c(brand.gis_icon_2)} by you",
```

Comment:
In this file you'll need to add the gis_icon_2 to the cityConfig:

1. Add a field [here](https://github.com/alexjball/adopt-a-drain/blob/ec47743366b896873547e3fae2e75c72d478b2be/app/views/main/index.html.haml#L71C1-L71C1) 
2. That makes it available in the [cityConfig global](https://github.com/alexjball/adopt-a-drain/blob/ec47743366b896873547e3fae2e75c72d478b2be/app/assets/javascripts/main.js.erb#L4) (used [here](https://github.com/alexjball/adopt-a-drain/blob/ec47743366b896873547e3fae2e75c72d478b2be/app/assets/javascripts/main.js.erb#L8C46-L8C46) for example)
3. reference it in javascript like 

```
name: `${cityConfig.gisIcon2} by you`
```

Reply:
so I tried this and got an error "ActionView::Template::Error (key not found: :gis_icon_1".  It looks like a key has to be added to a schema or model somewhere.

![image](https://github.com/alexjball/adopt-a-drain/assets/67333843/d1ec04d9-7bc0-43c1-8a07-64989a3f820c)

adopt-a-drain-web-1  |   
adopt-a-drain-web-1  | ActionView::Template::Error (key not found: :gis_icon_1):
adopt-a-drain-web-1  |     70:   this.cityConfig = { 
adopt-a-drain-web-1  |     71:     mapCenterLat: #{c('city.center.lat')}, 
adopt-a-drain-web-1  |     72:     mapCenterLng: #{c('city.center.lng')},
adopt-a-drain-web-1  |     73:     gis_icon_1: #{c('brand.gis_icon_1')},
adopt-a-drain-web-1  |     74:     gis_icon_2: #{c('brand.gis_icon_2')},
adopt-a-drain-web-1  |     75:     gis_icon_3: #{c('brand.gis_icon_3')}  
adopt-a-drain-web-1  |     76:   }

Reply:
or more likely "brand" needs to be explicitly imported or referenced somewhere